unit11/unit11.js

```diff
@@ -0,0 +1,121 @@
+// Exercise : 1 
+// method 1
+const repeatCharMethodOne = (char, number) => {
+  const arrayChar = []
+  for (let i = 0; i < number; i++) {
+    arrayChar.push(char)
+  }
+  return arrayChar
+}
+// method 2 
+const repeatCharMethodTwo = (char, number) => {
+  const arrayChar = []
+  let i = 0 
+  while (i < number) {
+    arrayChar.push(char)
+    i++
+  }
+  return arrayChar
+}
+
+// Exercise 2 
+const getReverseArray = (array = []) => {
+  const reverseArray = []
+  for (let i = array.length - 1; i >= 0; i--) {
+    reverseArray.push(array[i])
+  }
+  return reverseArray
+}
+
+// Exercise 3 
+const deleteFalseValue = (array = []) => {
+  const truValue = []
+  for (let i = 0; i < array.length; i++) {
+    if (array[i] === 0) continue
+    if (typeof(array[i]) === 'number') {
+      truValue.push(array[i])
+    }
+  }
+  return truValue
+}
+
+// Exercise 4 
+const createObject = (array = []) => {
+  const newObject = {}
+  array.forEach(item => {
+    newObject[item[0]] = item[1]
+  })
+  return newObject
+}
+
+// Exercise 5 
+const sortNumber = (array = []) => {
+  // return array.sort()
+  for (let i = 0; i < array.length - 1; i++) {
+    for (let j = i + 1; j < array.length; j++) {
+      if (array[i] > array[j]) {
+        let tg = array[i]
+        array[i] = array[j]
+        array[j] = tg   
+      }
+    }
+  }
+  return array
+}
+
+// Exercise 6 
+const checkObject = a => {
+  if (typeof(a) !== 'object') return false
+  if (Array.isArray(a) === true) {
```

Comment:
Cái gì viết rối thế này?
sao ko viết thành thế này cho gọn:

```suggestion
  return typeof(a) === 'object' && !Array.isArray(a)
```

Nhưng vẫn sai
Sẽ như thế nào khi nhập:
`checkObject(null)`